src/audio/graph.rs

```diff
@@ -2,26 +2,46 @@ use audio::node::{AudioNodeMessage, AudioNodeType};
 use audio::render_thread::{AudioRenderThread, AudioRenderThreadMsg};
 use std::sync::atomic::{AtomicUsize, Ordering, ATOMIC_USIZE_INIT};
 use std::sync::mpsc::{self, Sender};
+use std::sync::{Arc, RwLock};
 use std::thread::Builder;
 
 static NEXT_NODE_ID: AtomicUsize = ATOMIC_USIZE_INIT;
 
 pub struct AudioGraph {
     sender: Sender<AudioRenderThreadMsg>,
+    sample_rate: f32,
+    current_time: Arc<RwLock<f64>>,
```

Comment:
does the AudioGraph need this? I'd rather have this be owned by the actual render thread without an RWLock

Reply:
overall we should clean up the mutability story here but I have a plan for that

Reply:
The WebAudio API exposes sync getters for these values, so I figured that we should make the control thread own them. I guess we can have a sync channels with buffer size 0 for these getters (?).